filament/src/details/Scene.h

```diff
@@ -180,6 +183,8 @@ class FScene : public Scene {
     tsl::robin_set<utils::Entity> mEntities;
     RenderableSoa mRenderableData;
     LightSoa mLightData;
+    uint32_t mUboSize = 0;
+    filament::Handle<HwUniformBuffer> mUniformBufferHandle;
```

Comment:
This could perhaps have a one-line comment describing what data is stored in this particular uniform buffer.